src/kv/kv.h

```diff
@@ -899,16 +899,14 @@ namespace kv
           throw std::logic_error(
             "Transaction must be over maps in the same store");
       }
-      else
+
```

Comment:
this is needed because if the `view_list` is already populated from deserialising but not comiting, then any reads for any views will be out of date